juriscraper/opinions/united_states/state/lactapp_4.py

```diff
@@ -0,0 +1,139 @@
+"""Scraper for Louisiana Court of Appeal, fourth Circuit
+CourtID: lactapp_4
+Court Short Name: La. Ct. App. 4th Cir.
+Author: Luis-manzur
+History:
+  2025-04-22: Created by Luis-manzur
+"""
+
+import re
+from datetime import date, datetime
+
+from juriscraper.AbstractSite import logger
+from juriscraper.lib.date_utils import unique_year_month
+from juriscraper.OpinionSiteLinear import OpinionSiteLinear
+
+
+class Site(OpinionSiteLinear):
+    first_opinion_date = datetime(2019, 7, 17)
+    days_interval = 28  # ensure a tick for each month
+    date_regex = re.compile(r"\d{2}/\d{2}/\d{4}")
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.court_id = self.__module__
+        self.url = "https://www.la4th.org/Default.aspx"
+        self.search_is_configured = False
+        self.target_date = datetime(2025, 1, 1)
+        self.make_backscrape_iterable(kwargs)
+        self.status = "Unknown"
```

Comment:
I  actually think all of these opinions should be categorized as published.  I see nothing here to suggest they are unpublished, or that it's a mixed bag.